### ISSUE TITLE
Aetherslate Cave missing

### ISSUE THREAD

**Issue by Zonk1987** [FTBTeam/FTB-Modpack-Issues#4372]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.8.0

### Describe the bug

I flew through my world to find an Aetherslate Cave using Nature's Compass. I flew several 10k blocks and found nothing.
(World is Updated from 1.7.0->1.8.0 and i found the new added Botania Flowers so it should be freshly generated bioms)

I also created a new creativ world and teleported 10k of blocks around to see i there is somewhere a Cave but never found one.

In the Nature's Compass if you are looking up the Aetherslate Cave there is also no Dimension bound to.

### Steps to reproduce

Steps are in the Discription.

### Expected behaviour

When i use the Nature's Compas i should find the structure

### Screenshots

![2023-11-30 05_22_25-Window](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/25836879/6e687163-da8d-4397-9a72-2ff5d46e1cec)

**Comment by Zonk1987** [comment]
Yeah you are right but isn't it then the Aestersteel Mod totally useless and can be removed from the pack ?

**Comment by Klawn** [comment]
i guess you should be able to get some aethersteel in some chest ? Again, i don't know, i'm just trying to figure this out with you. For what i see, the generation of the ore from aethersteel was annoying and it seems that even the author of the mod is thinking about removing them... But yeah, if you can't get them via chest, i don't know what is the purpose of these mods.

**Comment by Klawn** [comment]
https://github.com/FTBTeam/FTB-Modpack-Issues/issues/4165#issuecomment-1799400463

Here is your answer. ;)
